build.sbt

```diff
@@ -567,7 +567,20 @@ lazy val junit = project.in(file("test") / "junit")
     javaOptions in Test += "-Xss1M",
     libraryDependencies ++= Seq(junitDep, junitInterfaceDep, jolDep),
     testOptions += Tests.Argument(TestFrameworks.JUnit, "-a", "-v"),
-    // testFrameworks -= new TestFramework("org.scalacheck.ScalaCheckFramework"),
+    unmanagedSourceDirectories in Compile := Nil,
+    unmanagedSourceDirectories in Test := List(baseDirectory.value)
```

Comment:
Yes but if you look at the new `scalacheck` project, it also contains these two lines, in white. It's a diffing artifact that those in `junit` appear green and those in `scalacheck` appear white. Logically speaking, the ones in `scalacheck` should be green.
In other words, this looks good to me.